mantidproject/mantid
# Increase tolerance on ISIS Powder HRPD test for OSX

**Requires testing on OSX**

Tolerance for `FocusTest` in `ISIS_PowderHRPDTest` was increased to allow for differences in splines on OSX.

**To test:**

Make sure `ISIS_PowderHRPDTest` passes locally and on the build server

Fixes #21795 

**Release Notes** 
Internal change, does not need to be in the release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
